.circleci/config.yml

```
@@ -0,0 +1,73 @@
+skip-check: &skip-check
+  name: Check for [ci skip]
+  command: bash .circleci/early_exit.sh
+
+apt-run: &apt-install
+  name: Install apt packages
+  command: |
+    sudo apt update
+    sudo apt install -y graphviz build-essential
+
+tox-install: &tox-install
+  name: Install Tox
+  command: |
+    sudo pip install tox codecov
+
+version: 2
+jobs:
+  egg-info-37:
+    docker:
+      - image: circleci/python:3.7
+    steps:
+      - checkout
+      - run: python setup.py egg_info
+
+  pycodestyle:
+    docker:
+      - image: circleci/python:3.7
+    steps:
+      - checkout
+      - run: sudo pip install pycodestyle
+      - run: pycodestyle --count
+
+  tests:
+    docker:
+      - image: circleci/python:3.7
+    steps:
+      - checkout
+      - run: *skip-check
+      - run: *apt-install
+      - run: *tox-install
+      - run: tox -e py37
+      - run: codecov
+
+  html-docs:
+    docker:
+      - image: circleci/python:3.7
+    steps:
+      - checkout
+      - run: *skip-check
+      - run: *apt-install
+      - run: *tox-install
+      - run: tox -e py37-build_docs
+      - store_artifacts:
+          path: docs/_build/html
+
+      - run:
+          name: "Built documentation is available at:"
+          command: DOCS_URL="${CIRCLE_BUILD_URL}/artifacts/${CIRCLE_NODE_INDEX}/${CIRCLE_WORKING_DIRECTORY/#\~/$HOME}/docs/_build/html/index.html"; echo $DOCS_URL
+
+workflows:
+  version: 2
+
+  egg-info:
+    jobs:
+      - egg-info-37
+
+  pycodestyle:
+    jobs:
+      - pycodestyle
+
+  test-documentation:
+    jobs:
+      - html-docs
```

```suggestion
      - html-docs

  tests:
    jobs:
      - tests 

```
Should make the final test run on circle.